### PR TITLE
DDCNL-1336

### PR DESCRIPTION
**Pull request by oc-ste**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-coffeescript" % "1.0.2")
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 
-addSbtPlugin("org.brianmckenna" % "sbt-wartremover" % "0.11")
+//addSbtPlugin("org.brianmckenna" % "sbt-wartremover" % "2.4.5")
```
Comment (jpatton638) [inline project/plugins.sbt:11]: Delete please

Comment (oc-ste) [reply project/plugins.sbt:11]: Deleted

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -6,13 +6,12 @@ object AppDependencies {
...
   val compile = Seq(
     ws,
-    "uk.gov.hmrc"         %% "play-breadcrumb"          % "1.0.0",
     "uk.gov.hmrc"         %% "bootstrap-play-26"        % "1.5.0",
     "uk.gov.hmrc"         %% "play-partials"            % "6.9.0-play-26",
     "uk.gov.hmrc"         %% "url-builder"              % "3.3.0-play-26",
     "uk.gov.hmrc"         %% "http-caching-client"      % "9.0.0-play-26",
     "uk.gov.hmrc"         %% "play-language"            % "4.2.0-play-26",
-    "uk.gov.hmrc"         %% "local-template-renderer"  % "2.8.0-play-26",
+    "uk.gov.hmrc"         %% "local-template-renderer"  % "2.8.1-play-26",
```
Comment (anand462) [inline project/AppDependencies.scala:14]: The latest version is 2.9.0-play-26. Please update the same.

Comment (oc-ste) [reply project/AppDependencies.scala]: Done

**Pull request merged**